test/gtest/ucp/test_ucp_tag_offload.cc

```diff
@@ -374,9 +374,6 @@ UCS_TEST_P(test_ucp_tag_offload, eager_multi_probe,
 {
     activate_offload(sender());
 
-    // TODO: Update length to bigger value so that multi-fragment eager is used
-    //       when multi-eager offload probe issue is fixed (like below).
-    // size_t length = ucp_ep_config(sender().ep())->tag.rndv.am_thresh.remote - 1;
```

Comment:
seems the length did not really change.. so how did it work before the fix?

Reply:
`short_progress_loop` was too long), so that all fragments arrived before the probe